src/components/ValueSet/CodingField.tsx

```diff
@@ -83,7 +91,7 @@ export const CodingField = ({
               <Input
                 {...field}
                 placeholder={t("unverified")}
-                className={!field.value ? "text-gray-500" : undefined}
+                className={cn(!field.value ? "text-gray-500" : undefined)}
```

Comment:
using `cn` is nice for conditional classes, but `cn` comes with a slight performance overhead. It's good practice to skip using for places where that's absolutely not needed.

`cn` is best used when you have multiple classes that could potentially need a smart way to resolve which classes would get preference when similar classes are present from multiple conditions.

I'd always avoid using it when it's only if multiple sets of classes are being applied. for example `cn("default classes", conditional classes)`

```suggestion
                className={!field.value && "text-gray-500"}
```